bitnami/mongodb/Chart.yaml

```diff
@@ -26,4 +26,5 @@ name: mongodb
 sources:
   - https://github.com/bitnami/bitnami-docker-mongodb
   - https://mongodb.org
-version: 10.6.1
+version: 10.6.2
+
```

Comment:
could you delete this blank line?

Reply:
Fixed

Reply:
HI, thank you for all the feedback, I hope you are happy with these fixes, please let me know if you would like me to change anything else.